crypto/wolfcrypt/shim/rubin_wc_shim.c

```
@@ -76,35 +106,41 @@ int32_t rubin_wc_verify_mldsa87(const uint8_t* pk, size_t pk_len,
     if (pk == NULL || sig == NULL || digest32 == NULL) {
-        return -10;
+        return RUBIN_WC_ERR_ML_DSA87_NULL;
+    }
+    if (pk_len != RUBIN_ML_DSA87_PUBKEY_BYTES) {
+        return RUBIN_WC_ERR_ML_DSA87_PK_LEN;
+    }
+    if (sig_len != RUBIN_ML_DSA87_SIG_BYTES) {
+        return RUBIN_WC_ERR_ML_DSA87_SIG_LEN;
     }
 
     int rc;
```

Clang (14.0.6)

[warning] variable 'rc' is not initialized

(cppcoreguidelines-init-variables)